README.md

```diff
@@ -75,6 +75,39 @@ requestBody := gsclient.IPCreateRequest{
 client.CreateIP(ctx, requestBody)
 ```
 
+For updating/scaling server resources you could use:
+
+```go
+myServerUuid := "[Server UUID]"
+backgroundContext := context.Background()
+
+// No hotplug available for scaling resources down, shutdown server first via AHCI
```

Comment:
Shouldn't this read "ACPI" not "AHCI" (AHCI is Advanced Host Controller Interface and typically refers to the SATA PCI bridge...doesn't it?)

Reply:
:man_facepalming: it's ACPI for sure